example/integration_test/test_utils.dart

```diff
@@ -0,0 +1,77 @@
+import 'dart:typed_data';
+import 'package:base_x/base_x.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:ledger_cardano/ledger_cardano.dart';
+import 'package:ledger_cardano/src/models/parsed_address_params.dart';
+import 'package:ledger_flutter/ledger_flutter.dart';
+import 'package:ledger_cardano/src/operations/complex_ledger_operations.dart';
+import 'package:ledger_cardano/src/utils/utilities.dart';
+
+import 'get_extended_public_key_test_cases.dart';
+
+String addressHexToBase58(String addressHex) {
+  final base58 = BaseXCodec('123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz');
+  final bytes = hex.decode(addressHex);
+  return base58.encode(Uint8List.fromList(bytes));
+}
+
+String base58ToHex(String base58Address) {
+  final base58 = BaseXCodec('123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz');
+  final bytes = base58.decode(base58Address);
+  return bytes.map((byte) => byte.toRadixString(16).padLeft(2, '0')).join();
+}
+
+Future<String> deriveAddress(
+    CardanoLedgerApp cardanoApp, LedgerDevice device, CardanoNetwork network, ParsedAddressParams params) async {
+  final operation = CardanoDeriveAddressOperation(
+    params: params,
+    network: network,
+  );
+
+  final addressResult = await cardanoApp.ledger.sendComplexOperation<String>(
```

Comment:
We should only test stuff accessible via `cardanoApp`'s api. We should not even expose `ledger` to allow "custom" operations to be made.

P.s. If there are stuff that you want to test and you cannot, you may want to change/add new operations to `cardanoApp` directly.